src/cmem/markdown/Markdown.tsx

```diff
@@ -151,22 +154,24 @@ export const Markdown = ({
                         }}
                         children={String(children).replace(/\n$/, "")}
                         language={match[1]}
+                        target={linkTargetName}
                     />
                 ) : (
-                    <code {...rest} className={className}>
+                    <code {...rest} target={linkTargetName} className={className}>
```

Comment:
There is no `target` attribute on the `code` element in HTML. See https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes